netneurotools/metrics.py

```diff
@@ -99,3 +101,87 @@ def communicability_wei(adjacency):
     cmc[np.diag_indices_from(cmc)] = 0
 
     return cmc
+
+
+def rich_feeder_peripheral(x, sc, stat='median'):
+    """
+    Calculates median or mean "connectivity" in rich (hub to hub), feeder
+    (hub to non-hub), and peripheral (non-hub to non-hub) links.
```

Comment:
The first line of the doc-string should always be one line. Perhaps this can be reduced to:

```suggestion
    Calculates connectivity values in rich, feeder, and peripheral edges
```

Since you have the slightly longer description below we can avoid repeating ourselves too much!